packages/wallet/ui/src/util/WalletBackendAdapter.js

```diff
@@ -202,8 +202,17 @@ export const makeWalletBridgeFromFollower = (
           const { status } = updateRecord;
           console.log('offerStatus', { status, offers });
           const oldOffer = offers[status.id];
-          if (
-            oldOffer &&
+          if (!oldOffer) {
+            break;
```

Comment:
here we have an offerStatus update for an unknown offer. I think we'd benefit from `console.warn` that this happened

Reply:
Done